mackerel-plugin-redis/lib/redis.go

```diff
@@ -106,12 +94,12 @@ func (m RedisPlugin) MetricKeyPrefix() string {
 // FetchMetrics interface for mackerelplugin
 func (m RedisPlugin) FetchMetrics() (map[string]interface{}, error) {
 	network := "tcp"
-	target := fmt.Sprintf("%s:%s", m.Host, m.Port)
+	address := fmt.Sprintf("%s:%s", m.Host, m.Port)
```

Comment:
[nit] is better to use `net.JoinHostPort`

Reply:
Make sense 👍